package.json

```diff
@@ -72,37 +72,38 @@
     "knockout-validation": "Knockout-Contrib/Knockout-Validation#^2.0.4",
     "leaflet": "0.7.7",
     "less": "~3.10.3",
-    "npm": "^6.14.5",
-    "uglify-js": "^2.6.1",
     "markdown-it": "7.0.1",
     "mocha": "^2.3.3",
     "moment": "2.12.0",
     "multiselect": "0.9.12",
+    "npm": "^6.14.5",
     "nprogress": "0.2.0",
     "nvd3": "1.1.10",
     "nvd3-1.8.6": "npm:nvd3#1.8.6",
     "perfect-scrollbar": "^1.5.0",
+    "phantomjs-prebuilt": "^2.1.16",
```

Comment:
What are we still using this for? Also, I see it's in "dependencies" and not "devDependencies"—is this used by deployed code?

Reply:
our `xpath` util depends on it, and if allowed to install `phantomjs` on its own, it picks a package that fails due to proxy errors... so until we get rid of that dependency in `xpath`, we have to include this so that the deploy doesn't fail.

Reply:
dang i spoke to soon. we might have to pre-install XPath on machines before the deploy process as I'm now getting
```
 out: info There appears to be trouble with your network connection. Retrying...
out: error An unexpected error occurred: "https://registry.yarnpkg.com/phantomjs-prebuilt/-/phantomjs-prebuilt-2.1.16.tgz: tunneling socket could not be established, cause=getaddrinfo EAI_AGAIN none".
 out: info If you think this is a bug, please open a bug report with the information provided in "/home/cchq/www/staging/releases/2020-07-02_12.41/yarn-error.log".
out: info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```